features/rest_api.feature

```diff
@@ -30,7 +30,7 @@ Feature: REST API
       """
     And I write to "features/step_definitions/post_steps.rb" with:
       """
-      When('the client requests GET /{word}') do |path|
+      When('the client requests GET \/{word}') do |path|
```

Comment:
(Curious: Is the unrelated to the Ruby 3 thing?)

Reply:
It is not

I unfortunately create the branch from another one instead of main.

The other branch has been merged into main already

Reply:
I'll rebase properly soon

Reply:
Cool, no rush for my sake